superset/key_value/schemas.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from marshmallow import fields, Schema
+
+
+class KeyValuePostSchema(Schema):
+    value = fields.String(
+        required=True, allow_none=False, description="Any type of JSON supported value."
+    )
```

Comment:
Do we want to enforce some certain type here? I wonder if we should force this to be a `fields.Dict` to make sure all values are JSON objects. This would make sure the frontend never has to at least worry about receiving something that's unparseable.

Reply:
Since this model is generic I prefer to not enforce the type. We could have cases where we want to store strings or numbers. One example would be to store a textual draft.